docs/components/index.md

```diff
@@ -67,18 +67,21 @@ In the following example, the `ParentComponent` sets the value of the `Title` pr
 
 ## Child content
 
-Components can set the content of an element in another component by assigning to a property that represents a [RenderFragment](/api/Microsoft.AspNetCore.Blazor.RenderFragment.html) type.
-
-In the following example, the `ChildContent` element of the `ParentComponent` sets the value of the `ChildComponent`'s `ChildContent` property. `ChildContent` is a `RenderFragment` type and supplies the content to the Bootstrap panel.
+Components can set the content of an element in another component. The assigning component provides the content in the tags that specify the receiving component. For example, a `ParentComponent` can provide content that is to be rendered by a `ChildComponent` by placing the content inside **\<ChildComponent>** tags.
```

Comment:
This doesn't sound quite right to me. Child content allows a parent component to capture rendered content from a child component and then use it as part of its own rendering. I don't think the parent component provides content to the child content. I think it's the the other way around.

Reply:
High level that's pretty close. @SteveSandersonMS can speak better to the actual mechanics of how it works, but the high level the idea is that if you have a parent component with child content, like this:

```html
<Parent>
    // Child content with HTML, components, fancy Razor syntax, etc.
</Parent>
```

then the Parent component can capture the child content in a `RenderFragment` using a `ChildContent` property and then render it as part of the Parent component's rendered content.

Reply:
I totally don't have it that way today ... and I find that somewhat confusing. The parent is a whole component. The child is a whole component. What I did was stick the child tags into the parent and then describe it based on how it's composed. Today, it goes like this ...

*ParentComponent.cshtml*:

```cshtml
@page "/ParentComponent"

<h1>Parent-child example</h1>

<ChildComponent Title="Panel Title from Parent">
    Panel content of the child component is supplied by the parent component.
</ChildComponent>
```

*ChildComponent.cshtml*:

```cshtml
<div class="panel panel-success">
  <div class="panel-heading">@Title</div>
  <div class="panel-body">@ChildContent</div>
</div>

@functions {
    public string Title { get; set; }
    public RenderFragment ChildContent { get; set; }
}
```

Reply:
I guess it boils down to how this is described and the sequence of what goes where and when ...

> have a parent component with child content

I didn't see it quite that way. The parent component only has the child content by virtue of the fact that the content is *inside of the tags that specify the child component.* If the child just looks up into the parent to pull that content down into its `RenderFragment` ... that's cool. Then, the child becomes fully rendered for the parent. I think the order is easy to state (if that order I wrote out is correct).

The greater challenge is making sure the reader doesn't get confused on what "child content" means. It's not the whole rendered child component. It's just the content inside the tags for the child component that end up in the `RenderFragment`. I'm assuming I'm correct about that. It was written and approved that way when we merged the topic.

I'm going to hold off on further changes until @SteveSandersonMS has a chance to look this over and say what's best.